### DISCUSSION TITLE
Allow users to mark tracks as maxed driver level

### DISCUSSION THREAD

**Discussion by DragRedSim** [MKT-Playbook/Hub#13] [Feature requests]
Each individual course has a limit on the amount of driver level percentage it will add to your driver level (in my estimation, it’s somewhere around 1.15 to 1.2 levels per course). It goes up with podium finishes, down with finishes below fourth; but after you’ve drained a course’s level points, it won’t boost your level any further. This is static across tours; if the same course appears in two tours, but you completed it in the first, you can’t earn any more driver level points from it in the second tour. It would be nice if we could mark each track as having been completed, so that we can focus our efforts on tracks which have not given our driver level all they can to help boost our overall scoring potential.

**Comment by TitusPosiadly** [comment]
Are you referring to the XP gained for the player level?

**Comment by DragRedSim** [comment]
Yes, I was. I see this has been implemented in the latest version of the app; it’ll make it easier to determine which courses are best for playing to increase player level, useful since that adds to your score on all tracks.

One small glitch I noticed, though; if you select a track to edit its “completed” status from the Track menu, then choose a different variant (normal, R, T, R/T), the checkbox doesn’t update to the state of the variant selected; it will show the setting it had previously, or if it hadn’t been changed, that of the normal variant of the track (since that’s the default one opened). It won’t be an issue for many since it’s something people will only need to do once (to set it up with their current data), but it could be a bit confusing. For example:

- Select track (let’s just say NYM as an example)
- I completed this track’s XP last tour, so I tick that off
- I also completed NYM T, so I change the track to the Trick variant
- The checkbox does not clear, even though I haven’t previously marked it as cleared

**Reply by TitusPosiadly** [reply]
Hey! Yes, thanks for letting me know. I had this issue reported by another user just the other day and was already able to fix it. I do however need a couple more days to fix some more unrelated bugs before I can upload the new version to the AppStore. I will create a new issue for this nonetheless though.
